### DISCUSSION TITLE
Please tell us about your use cases!

### DISCUSSION THREAD

**Discussion by ljusten** [google/cdc-file-transfer#85] [Q&A]
We're super curious to know whether you find these tools useful - or why not, and what you use them for.

Did you try cdc_rsync or cdc_stream?
Did you use them and they worked like a charm?
Did you try using them, but they didn't work?
Do you have no idea what this page is about?

In any case, please tell us! We need your feedback to understand what the tools are useful for and where they lack.

Thanks!

**Comment by pcc** [comment]
I plan to use this for Android development to speed up syncing arm64 Android images between my main workstation and my arm64 Linux machine running an Android emulator ([these instructions](https://github.com/pcc/llvm-project/blob/apple-pac5/patches/README.cuttlefish) that I previously put together use scp to copy the images). Android images are typically several GB large and there are typically very few changes after each rebuild, so this seems like an ideal use case for something like CDC. I run Linux on my main workstation so this will require Linux client support.
